### PR TITLE
Adding DIRECTORY_PATH instructions in README.md

### PR DESCRIPTION
**Pull request by Raimmaster**
Adding to the README.md so people know it can be used and why.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Thus `https://[gitlab.domain/org/dependabot-script-repo]/pipeline_schedules` das
 * Set in the schedule the required variables:
   * `PROJECT_PATH`: `group/repository`
   * `PACKAGE_MANAGER_SET`: `bundler,composer,npm_and_yarn`
+* If you'd like to specify the directory that contains the package list in the repository, you can set the following environment variable:
```
Comment (greysteil) [inline README.md]:
```suggestion
* If you'd like to specify the directory that contains the manifest file in the repository, you can set the following environment variable:
```

Comment (Raimmaster) [reply README.md]: Thank you. I've changed the wording as suggested.

**Comment by greysteil** [comment]
Awesome, thanks!

**Comment by Raimmaster** [comment]
You're welcome, and thank you again!

**Pull request merged**